proposed/0026-dynamic-data-support/0026-dynamic-data-support.md

```diff
@@ -0,0 +1,153 @@
+- Feature Name: Ability to Support Dynamic Data Handling
```

Comment:
Perhaps a better title is "append by all" structured data type as there will be several types such as whitelisted append SD etc. as well as private append only etc. the network will handle.  A small bit at a time is good though.